lib/SDIR/Translate/translateToSDFG.cpp

```diff
@@ -731,6 +749,13 @@ LogicalResult translation::translateToSDFG(AllocOp &op, JsonEmitter &jemit) {
   }
   jemit.endList(); // shape
 
+  jemit.startNamedList("offset");
```

Comment:
do we need this default?

Reply:
Yes a test fails otherwise. I'll provide all the failing defaults

Reply:
Thank you. We can remove those later.